pandas/core/resample.py

```diff
@@ -519,21 +519,64 @@ def nearest(self, limit=None):
 
     def backfill(self, limit=None):
         """
-        Backward fill the values
+        Backward fill the values.
+
+        In statistics, imputation is the process of replacing missing data with
+        substituted values [1]_. When resampling data, missing values may
+        appear (e.g., when the resampling frequency is higher than the original
+        frequency). The backward fill will replace NA values with the next
+        non-NA value in the sequence.
 
         Parameters
         ----------
         limit : integer, optional
-            limit of how many values to fill
+            Limit of how many values to fill.
 
         Returns
         -------
-        an upsampled Series
+        Series
+            An upsampled Series with backward filled NA values.
 
         See Also
         --------
-        Series.fillna
-        DataFrame.fillna
```

Comment:
can you add a Resampler.pad, nearest, and fillna refs @jorisvandenbossche how do we reference these exactly here?

Reply:
It might be you can simply refer to them as 'pad', 'nearest', .. because they live on the same class. But need to check.

Reply:
If I run `python make.py --single pandas.core.resample.Resampler.backfill`, the links are never created. But if I run `python make.py html`, the links are created correctly if I use: `pandas.Series.fillna` (`Series.fillna` and `pandas.core.series.Series.fillna` don't work) and `fillna` or `Resampler.fillna`.